src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -2,6 +2,13 @@
 
 from collections import ChainMap
 from pathlib import Path
+from platform import (
+    node as platform_node,
+    platform as platform_platform,
+    release as platform_release,
+    system as platform_system,
+    version as platform_version,
+)
 from typing import Dict, List, Literal, Optional, Union
```

Comment:
Why just not:
`import platform
`
and then later:

`platform.platform` (instead of `platform_platform`) etc